bucket/dadroitjsonviewer.json

```
@@ -0,0 +1,35 @@
+{
+    "version": "1.1.1407",
+    "description": "A JSON viewer that gives a new approach to process JSON Data files.",
+    "homepage": "https://viewer.dadroit.com/",
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://viewer.dadroit.com/legal"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://viewer.dadroit.com/releases/win/Dadroit%20Viewer%201.1%20Build%201407%20x64%20Setup.exe",
+            "hash": "fda57d60f26aead25d26783e3508aee1f1fe1cbeb6f8456c5159659b34a57439"
+        }
+    },
+    "innosetup": true,
+    "bin": "DadroitViewer.exe",
+    "shortcuts": [
+        [
+            "DadroitViewer.exe",
+            "Dadroit JSON Viewer"
+        ]
+    ],
+    "checkver": {
+        "url": "https://viewer.dadroit.com/js/app.js",
+        "regex": "Ver (?<main>[\\d.]+) (?<build>\\d+)",
+        "replace": "${main}.${build}"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://viewer.dadroit.com/releases/win/Dadroit%20Viewer%20$matchMain%20Build%20$matchBuild%20x64%20Setup.exe"
```

No need to limit URL with $matches, which will prohibit user from installing older versions.

```suggestion
                "url": "https://viewer.dadroit.com/releases/win/Dadroit%20Viewer%20$majorVersion.$minorVersion%20Build%20$patchVersion%20x64%20Setup.exe"
```